owlify/src/test/java/org/archcnl/owlify/core/AbstractOwlifyComponentTest.java

```diff
@@ -0,0 +1,51 @@
+package org.archcnl.owlify.core;
+
+import static org.junit.Assert.*;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.apache.jena.rdf.model.Model;
+import org.junit.Test;
+
+public class AbstractOwlifyComponentTest extends AbstractOwlifyComponent {
```

Comment:
The big question that needs to be answered here is if we really want to write test cases for abstract classes. Testing them requires ugly workarounds like a test class that extends the tested class and method implementations for abstract methods that serve no purpose and do not belong into a test class.
Also what we really need to test are the classes that inherit from the abstract classes. They are the ones actually affecting the flow of the program. In this case, this would be the class FamixOntologyTransformer from owlify-famix which already has its own test class.

Reply:
I agree that we usually should not write tests for abstract classes.
In this case, the abstract class `AbstractOwlifyComponent` is not even needed, I've created a PR that removes it: https://github.com/Mari-Wie/ArchCNL/pull/188
So let's not write a test for it.

Reply:
Normally, I wouldn't have written those test either, but the ticket specifically requested that I write test for that abstract class. 
Nevertheless, your solution seems cleaner, therefor I will review your pull request.